actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -37,9 +37,6 @@ workflows:
             get_config:
               action: arteria-packs.get_pack_config
               publish:
-                hermes_base_url: <% task(get_config).result.result.hermes_base_url %>
```

Comment:
Maybe these variables can be removed from the config too?

Reply:
Aha. I thought I'd done so, but I've must have missed some.

Reply:
Oh, maybe I missed changes in the config. Sorry about that.